packages/protocol/contracts/common/interfaces/IEpochManager.sol

```diff
@@ -1,11 +1,8 @@
 // SPDX-License-Identifier: LGPL-3.0-only
-pragma solidity >=0.8.0 <0.9.0;
+pragma solidity >=0.5.13 <0.9.0;
 
-interface IBlocker {
-  function isBlocked() external view returns (bool);
-}
 
-interface IEpochManager is IBlocker {
+interface IEpochManager{
```

Comment:
- Add timestamp to initialize
- add startNextEpoch() and finishNextepcohProcess
- uncomment the rest or remove

Reply:
We are currently using getting  `block.timestamp` within the `initializeSystem` function and assigning it to the L2 epoch start time. 

were you thinking of also storing the timestamp of the last know block on L1 as  well?